app/components/Dashboard.tsx

```diff
@@ -108,8 +115,22 @@ const Dashboard: React.FC<Props> = ({ query: queryKey }) => {
         </section>
         {isAdmin && (
           <section>
-            <h3 id="admin-heading">Administration</h3>
-            <nav aria-labelledby="admin-heading"></nav>
+            <div id="admin-heading">
+              <h3>Administration</h3>
+            </div>
+            <nav aria-labelledby="admin-heading">
+              <Link passHref href={getMetabaseRoute()}>
+                <a target="_blank">Data Insights (Metabase)</a>
```

Comment:
Why `<a>` vs`<BCGovLink>` like elsewhere? (question, not suggestion, I don't know much about `<BCGovLink>`)

Reply:
Good point, thanks for noticing that. I don't know much about that too but it's better to be consistent and use that rather than `<a>` tag.
The only difference I noticed is that `BCGovLink` doesn't get purple if you visit that link.